### ISSUE TITLE
Better way to remove VPN Icon

### ISSUE THREAD

**Issue by Ryan-Goldstein** [AdguardTeam/AdguardForiOS#1011]
I'm proposing an option to remove the VPN icon when using Split Tunnel. I understand that this wasn't possible back in iOS 9, hence similar requests being marked as WontFix a few years ago, but it should be possible now. An app I've been using for a while that also uses Split Tunnel and has an option to hide the VPN icon is DNSCloak: https://itunes.apple.com/in/app/dnscloak-secure-dns-client/id1452162351

I prefer to only have this icon visible when actually using a Personal VPN (like ProtonVPN), which can be used alongside DNSCloak or AdGuard (using Split Tunnel).

Thanks!

**Comment by ameshkov** [comment]
We'll see if that's possible, thank you for the example!

**Comment by iDjay** [comment]
Any news on this one? :)

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by ameshkov** [comment]
You know, it seems that the method that was used for removing VPN icon no more works on iOS 13

**Comment by iDjay** [comment]
@ameshkov but hiding with the full tunnel option still works? Is it possible to add one for the split tunnel option?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by ameshkov** [comment]
@iDjay tbh, I don't know yet:) We have a lot to fix before we can get to this one.

**Comment by Ryan-Goldstein** [comment]
Thanks for the update, @ameshkov!

FYI, DNSCloak on iOS 13 public beta 1 still has a working option to hide the VPN icon while connect on demand is enabled, including split tunnel (ProtonVPN works while DNSCloak is enabled).

Here's the source: https://github.com/s-s/dnscloak

**Comment by ameshkov** [comment]
We use the very same approach to hide the VPN icon btw. I guess I was misled and this way is still useable)

**Comment by AlikhanAliev** [comment]
https://share.goabstract.com/99a4c4c1-7d21-4b60-a22f-be380bf193c7?sha=a300566945e29599277befd951ea7de8e888c1ea

**Comment by ameshkov** [comment]
Just like DNSCloak we will need to run AG DNS as a DNS proxy. Assigned to v4.1, our DNS library should be able to do this.
